### PR TITLE
[ErrorHandler] Report overridden @final constants

### PR DESCRIPTION
**Pull request by fancyweb**
| Q             | A
| ------------- | ---
| Branch?       | 6.1
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | -
| Tickets       | https://github.com/symfony/symfony/issues/44268
| License       | MIT
| Doc PR        | -

See https://github.com/symfony/symfony/issues/44268 and https://wiki.php.net/rfc/final_class_const.

### CODE REVIEW

**Review by yceruto — APPROVED** [review]
LGTM!

File: src/Symfony/Component/ErrorHandler/DebugClassLoader.php [hunk 1]
```diff
@@ -468,8 +469,9 @@ public function checkAnnotations(\ReflectionClass $refl, string $class): array
         self::$finalMethods[$class] = [];
         self::$internalMethods[$class] = [];
         self::$annotatedParameters[$class] = [];
+        self::$finalConstants[$class] = [];
         foreach ($parentAndOwnInterfaces as $use) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/ErrorHandler/DebugClassLoader.php:473]: let's keep a blank line before

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Component/ErrorHandler/DebugClassLoader.php (hunk 1 above)
Comment (fancyweb) [reply src/Symfony/Component/ErrorHandler/DebugClassLoader.php:473]: There is no blank line currently :sweat_smile:

**Review by derrabus — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @fancyweb.

**Pull request merged**